spec/components/parameters/reports/flow/flowId.ts

```diff
@@ -0,0 +1,16 @@
+import { ParameterObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../../../utils/ref';
+
+const flowId: ParameterObject = {
+  name: 'flowId',
+  in: 'query',
+  required: false,
+  description: 'This is the flow identifier. You can acess your <a href="https://app.zenvia.com/home/chatbot" target="_blank"> Flow ID here.</a>',
```

Comment:
"access" tem dois "c"s. Mas eu acho que eu usaria "You can view your Flow IDs here."